data/Auld_2000/metadata.yml

```diff
@@ -0,0 +1,170 @@
+    latitude (deg): -33.654
+    longitude (deg): 151.259
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+  Site 2, KCNP:
+    latitude (deg): -33.654
+    longitude (deg): 151.259
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+  Site 3, KCNP:
+    latitude (deg): -33.654
+    longitude (deg): 151.259
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+  Site 4, KCNP:
+    latitude (deg): -33.654
+    longitude (deg): 151.259
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+  Site 5, Royal NP:
+    latitude (deg): -34.13
+    longitude (deg): 151.081
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+  Site 6, Royal NP:
+    latitude (deg): -34.13
+    longitude (deg): 151.08
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+  Site 7, near Maroota:
+    latitude (deg): -33.464
+    longitude (deg): 150.995
+    description: open heath to open forest
+    soil type: soils derived from Hawkesbury Sandstone
+contexts: .na
+traits:
+- var_in: Seed weight (mg)
+  unit_in: mg
+  trait_name: seed_dry_mass
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: unknown
+  methods: unknown
+- var_in: Proportion viable
+  unit_in: '{count}/{count}'
+  trait_name: seed_viability
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: viability_replicates
+  methods: For all species, levels of seed dormancy and viability were assessed in
+    an initial germination trial. These comprised four replicates of 25 seeds placed
+    in 9-cm-diameter petri dishes on Whatman Grade 2 filter paper with added distilled
+    water. For Persoonia pinifolia, sufficient seeds were only available for three
+    replicates of 10 seeds. Seeds that germinated were considered to be in a non-dormant
+    state. Seeds that did not germinate were tested for viability by the tetrazolium
+    test (Lakon 1949) and scored as viable if stained or if white and firm, except
+    for Grevillea spp. and Kunzea spp. Seeds of Grevillea spp. were scarified and
+    then allowed to germinate. If these seeds did not subsequently germinate they
+    were then tested for viability with tetrazolium. Seeds were scored as viable if
+    they were white and firm or if they stained under tetrazolium. Seeds of Kunzea
+    spp. were soaked in distilled water and examined under a binocular microscope.
+    Seeds were scored as viable if the endosperm was white and firm.
+- var_in: Fire response
+  unit_in: .na
+  trait_name: resprouting_capacity
+  entity_type: population
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: literature, expert knowledge
+- var_in: seed sample half-life (years)
```

Comment:
It isn't a new trait - too much "trait splitting". I think the others don't specify their actual metric, but it will be something conceptually similar to half-life - the others aren't recording when the first or last seed stops being viable either, although they might not have an explicit half-life modeled. But if they use bins, I think they are suggesting half-life. I had the sense that Tony simply thought "seedbank_longevity" was referring to half-life most of the time.